bigquery/v2/go.mod

```diff
@@ -0,0 +1,3 @@
+module cloud.google.com/go/bigquery/v2
```

Comment:
Might need to drop something in the go.work as well for this.

Reply:
added the entry, and tidying didn't appear to touch the checksums (expected since the v2 doesn't do anything yet).